client/packages/system/src/Manage/Preferences/api/utils.ts

```diff
@@ -1,63 +1,37 @@
 import {
   LocaleKey,
-  PreferenceKey,
   TypedTFunction,
   UpsertPreferencesInput,
 } from '@openmsupply-client/common';
-import { AdminPreferenceFragment } from './operations.generated';
-import { Dispatch, SetStateAction } from 'react';
 
 export const inputValidation = (
   input: Partial<UpsertPreferencesInput>,
   t: TypedTFunction<LocaleKey>,
-  warning: (msg: string) => () => void,
-  preferences?: AdminPreferenceFragment[],
-  setActionValid?: Dispatch<SetStateAction<boolean>>
+  warning: (msg: string) => () => void
 ): boolean => {
-  const thresholdResult = thresholdValidation(input, t, warning, preferences);
+  const thresholdResult = thresholdValidation(input, t, warning);
 
   // Combine results
   const isValid = thresholdResult; // && otherResult
 
-  setActionValid?.(isValid);
-
   return isValid;
 };
 
 const thresholdValidation = (
   input: Partial<UpsertPreferencesInput>,
   t: TypedTFunction<LocaleKey>,
-  warning: (msg: string) => () => void,
-  preferences?: AdminPreferenceFragment[]
+  warning: (msg: string) => () => void
 ) => {
   const inputFirstThreshold = input?.firstThresholdForExpiringItems?.[0]?.value;
   const inputSecondThreshold =
     input?.secondThresholdForExpiringItems?.[0]?.value;
 
-  // Second threshold should not exceed 30 days
-  if (inputSecondThreshold && inputSecondThreshold > 30) {
-    warning(t('label.second-threshold-exceeds-days'))();
-    return false;
-  }
-
-  const existingFirstThreshold = preferences?.find(
-    pref => pref.key === PreferenceKey.FirstThresholdForExpiringItems
-  )?.value;
-  const existingSecondThreshold = preferences?.find(
-    pref => pref.key === PreferenceKey.SecondThresholdForExpiringItems
-  )?.value;
-
-  const firstThreshold = inputFirstThreshold ?? existingFirstThreshold;
-  const secondThreshold = inputSecondThreshold ?? existingSecondThreshold;
-
-  // Second threshold should not be less than first threshold
+  // Thresholds should not exceed 30 days
   if (
-    firstThreshold != null &&
-    secondThreshold != null &&
-    secondThreshold !== 0 &&
-    secondThreshold < firstThreshold
+    (inputSecondThreshold && inputSecondThreshold > 30) ||
+    (inputFirstThreshold && inputFirstThreshold > 30)
   ) {
-    warning(t('label.second-threshold-is-less-than-first-threshold'))();
+    warning(t('label.threshold-exceeds-days'))();
```

Comment:
Halo, apologies, I know its still this is still in draft. But I peeked and noticed this. 

I left a comment on the issue: https://github.com/msupply-foundation/open-msupply/issues/9785#issuecomment-3549512831. But basically first threshold should be lower than the second